components/locid_transform/src/fallback/algorithms.rs

```diff
@@ -129,10 +129,20 @@ impl<'a> LocaleFallbackIteratorInner<'a> {
         if locale.script().is_none() {
             if let Some(region) = locale.region() {
                 let language = locale.language();
-                if let Some(script) = self.likely_subtags.lr2s.get_copied_2d(
+                let mut maybe_script = self.likely_subtags.lr2s.get_copied_2d(
```

Comment:
Please split out fallback algorithm changes

Reply:
There should be no fallback algorithm changes except for poking the hole for `visit_default_script`.